Character/Classes/Sorcerer/Sorcerer.xml

```diff
@@ -241,6 +243,8 @@
 				<name>Metamagic Option (4th)</name>
 				<text>At 17th level, you learn an additional metamagic option.</text>
 			</feature>
+		</autolevel>
+		<autolevel level="17" IL="IL">
```

Comment:
IL unneeded

Reply:
I argue that lvl 10 and 17 are exactly Repeated Inlined Lists, which is exactly IL's option definition.

Reply:
That is a fair point, they are repeated, I simply have not used them for the meta magic options. This is fine.